index.d.ts

```diff
@@ -33,7 +33,7 @@ export interface Options<
 	 *
 	 * @default new Map()
 	 */
-	readonly cache?: CacheStorage<CacheKeyType, ReturnType>;
+	readonly cache?: CacheStorage<CacheKeyType, { data: ReturnType; maxAge?: number; }>;
```

Comment:
Why is `maxAge` optional? It is not being set conditionally in the code.

Reply:
If you aren't using the `maxAge` option then this value is not necessary.

Reply:
With this definition, you're basically saying that `maxAge` _may_ be sometimes provided to the underlying API. This is, however, not the case. `maxAge` is _always_ set, disregarding of whether you use it or not.

When you implement a custom cache storage you don't actually have to use the `maxAge` value, it is however, always there.

Reply:
Okay, thanks for that explanation @BendingBender! Changing `maxAge` to required.